manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/application.yaml

```diff
@@ -28,7 +28,7 @@ spec:
 
   info:
     - name: Application Namespace
-      value: {{ .Release.Namespace }}
+      value: "{{ .Release.Namespace }}"
```

Comment:
is it for some special issue?
I think yaml string doesn't require "" but integer boolean wants.

Reply:
Yes, this is meant to make this file a valid yaml file, because `{{` is not yaml syntax.
I used `yq` to automate yaml file processing, it errors out if the file is not a valid yaml.

Reply:
yes, {{}} is not a yaml format but the helm. We may do a test for this point after you built out MKP images.

Reply:
Definitely, thanks for thinking about this!

Reply:
To wrap up, so wrapping `{{}}` with `"{{}}"` allows the syntax to be both valid yaml and helm template.